SPECS/rubygem-rexml/rubygem-rexml.spec

```diff
@@ -2,7 +2,7 @@
 %global gem_name rexml
 Summary:        REXML is an XML toolkit for Ruby
 Name:           rubygem-%{gem_name}
-Version:        3.2.7
+Version:        3.3.4
```

Comment:
The CVE seems to be a medium. Should we target the `main` branch ?

Reply:
I am closing this PR, will open new PR for main branch.

Reply:
New PR - https://github.com/microsoft/azurelinux/pull/10187